### ISSUE TITLE
Where can I find the codes of loss functions?

### ISSUE THREAD

**Issue by chhyun** [Microsoft/CNTK#1608]
As you already know, we use the loss function like below...
ex)  ce = CrossEntropyWithSoftmax(outnodes, outputlayer)

but my problem is, I need to take a look at the code of predefined loss functions. (especially CrossEntropyWithSoftmax)

cause I need to add some parameters to loss functions.

but I can't find the file that contains these predefined loss functions.

Where can I find the code file containing predefined loss functions?

**Comment by cha-zhang** [comment]
Example of defining your own loss function: 
https://github.com/Microsoft/FERPlus/blob/master/src/train.py

**Comment by sayanpa** [comment]
You can also define your own loss function in python.

Closing this issue since there was a suggestion and no further communication on this thread. Please reopen the issue with details if you need a follow up.
